src/components/Tree/helpers/index.ts

```diff
@@ -0,0 +1,6 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+export * from './getAcceptedTypes';
+export * from './getItemPositionInParent';
+export * from './getNextItemToFocus';
+export * from './getPreviousItemToFocus';
```

Comment:
IMO these should be different files and some of them combined. 
getAcceptedTypes -> dnd.ts
getItemPositionInParent -> position.ts
getNextItemToFocus, getPreviousItemToFocus -> focus.ts

or something similar in nature.

Reply:
Easier to find the methods IMO
What would be the benefit?